cranelift-codegen/meta/src/isa/x86/instructions.rs

```diff
@@ -487,5 +489,29 @@ pub(crate) fn define(
         .operands_out(vec![f]),
     );
 
+    let GV = &operand("GV", &entities.global_value);
+    let addr = &operand("addr", iWord);
+    let x = &operand("x", iWord);
```

Comment:
Looks like `iWord` is `32..64`: do you only want 64 here?

Reply:
Indeed